src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,9 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  const reversedString = number.toString().split("").reverse().join("");
+  const reversedDigits = reversedString
+    .split("")
+    .map((digit) => parseInt(digit, 10));
+  return reversedDigits;
```

Comment:
This works, but if you look carefully theres actually some unneeded logic here that could be removed. At the end of line 2 youre joining up the reversed array, then the first thing you do with it on line 4 is to split it again. You can remove this step which would give you
```
  const reversedString = number.toString().split("").reverse();
  const reversedDigits = reversedString.map((digit) => parseInt(digit, 10));
  return reversedDigits;
```
Since we're making things simpler, we can also just chain the methods on the first 2 lines here together, so when the formatters finished with that we get
```
  const reversedString = number
    .toString()
    .split("")
    .reverse()
    .map((digit) => parseInt(digit, 10));
  return reversedString;
```
Finally, we can remove the unnecessary variable creation and just return the result of this expression:
```
  number
    .toString()
    .split("")
    .reverse()
    .map((digit) => parseInt(digit, 10));
```
Does the exact same job, but is much cleaner